src/System.Management.Automation/engine/parser/token.cs

```diff
@@ -1119,17 +1124,19 @@ public static class TokenTraits
             /*              Command */ "command",
             /*               Hidden */ "hidden",
             /*                 Base */ "base",
+            /*            AtAtCurly */ "@@splat",
```

Comment:
I was thinking of `@` as a unary operator.

Thinking that way, one could write `@ @{...}` or `@@{...}` to mean the same thing, though the later would likely be preferred.

Given that, I wouldn't expect new tokens of the form `@@(` or `@$` or `@@{`.

Reply:
I would've put it into the `Punctuators` section, similar to `AtCurly` but because it's range for the `TokenKind` enum is already full, I just decided to put it at the end. Shall I put it into the operators section then?
If I understand correctly, you'd want me to add a TokenKind for `@` instead of `@@{`and starting from `@` figure out how to parse if it is followed by`@{`?